html_css/v2/main-course/custom-properties.md

```diff
@@ -0,0 +1,205 @@
+### Introduction
+
+Custom properties (also known as CSS variables) can be a really useful and powerful tool when writing our CSS files. In short, they allows us to reference a CSS value however many times we want throughout a file. How is this any different or better than just using the value, you ask?
+
+By using custom properties, instead of having to update every single instance of a specific value ("This shade of red is too light, let's change the shade on all seven of these selectors"), we only need to update a single instance: the custom property itself.
+
+Not only that, but we can even use the same custom property in different selectors, giving each instance of the property a different value. This is incredibly useful for creating themes, such as the dark and light themes you see on many websites these days.
```

Comment:
the wording is confusing here.

I think it might be clearer to say something like:

> It is even possible to redefine custom properties in different contexts, which is useful for creating themes.... etc.

Reply:
We should avoid "not only that" as well, if it's referencing a previous paragraph, it likely should be part of that paragraph. Otherwise, we should be very specific in what "that" is.